src/main/java/uk/gov/di/gpg45engine/matchers/ScoreMatcher.java

```diff
@@ -9,6 +9,6 @@ public static boolean equalsOrGreater(Score score, Score score2) {
     }
 
     public static boolean greater(Score score, Score score2) {
-        return score.getScoreValue() >= score2.getScoreValue();
+        return score.getScoreValue() > score2.getScoreValue();
```

Comment:
At some point, we're going to want documention with links and diagrams so that these sorts of changes can be more obviously thought about.

For now, I'm assuming you know what you're doing.